main.rb

```diff
@@ -3,7 +3,7 @@
 def main
   app = App.new
   puts 'Welcome to the School Library App!'
-  run_app(app)
+  app.run
 end
```

Comment:
- [ ] Please while making the changes requested by the previous reviewer, I am getting an error while running your application so ensure that you fix this :point_down: and make sure your fix the changes required by the previous reviewer :relaxed: 

![image](https://github.com/codeWithHyder/my-school-library/assets/92303891/3b3c69f6-9a14-42dc-9b33-b2738c520487)

Reply:
fixed the issue